ucdenver-dissertation.cls

```diff
@@ -427,6 +427,10 @@
   \vskip \postSskip \nopagebreak
 }
 
+% Jordan's fix for Roman/arabic issue
```

Comment:
Add comment to top of document instead of here.

Technically you should have made this change in the two env dotfiles, but ill merge this in and migrate the changes myself.